src/main/java/sirius/kernel/nls/NLS.java

```diff
@@ -1039,6 +1039,13 @@ private static String formatSpokenRecentDateWithTime(LocalDateTime givenDateTime
                 throw new IllegalArgumentException(fmtr("NLS.errInvalidDecimalNumber").set("value", value).format(), e);
             }
         }
+        if (Amount.class.equals(clazz)) {
```

Comment:
isn't this super confusing that null or "" return null instead of Amount.NONE?

Reply:
You are right, the alternative would be a workaround in sirius-search